docs/NativeModulesAndroid.md

```diff
@@ -18,7 +18,7 @@ This guide will use the [Toast](http://developer.android.com/reference/android/w
 We start by creating a native module. A native module is a Java class that usually extends the `ReactContextBaseJavaModule` class and implements the functionality required by the JavaScript. Our goal here is to be able to write `ToastAndroid.show('Awesome', ToastAndroid.SHORT);` from JavaScript to display a short toast on the screen.
 
 ```java
-package com.facebook.react.modules.toast;
+package com.example.modules.toast;
```

Comment:
@satya164 Well this is how I have been doing and perhaps seen many people as well. 

helloapp/android/app/src/main/java/com/helloapp/MainActivity.java

I write my ModuleClass.java and AnExampleReactPackageClass as siblings to MainActivity.Java and I had this problem. And If  I am not wrong this where I write all my java code. ?